lib/menu.rb

```diff
@@ -5,9 +5,7 @@ def initialize(dishes)
   end
 
   def display
-    dishes.map do |key, value|
-      "#{key}: £%.2f" % [value]
-    end.join(", \n")
+    dishes.map { |key, value| "#{key}: £%.2f" % [value] }.join(", \n")
```

Comment:
This seems like a very straightforward way to add the dishes (I used a completely separate class but think this is cleaner)